src/app/style.scss

```diff
@@ -21,6 +21,33 @@
   margin-bottom: 55px;
 }
 
+/* Start dummy page */
+.dummy-page {
+  justify-content: center;
+  text-align: center;
+}
+
+.dummy-page h3 {
+  color: #152935;
```

Comment:
This color is defined in polymath-ui scss file as $poly-blue-dark
We should import colors from there

BUT
Probably we should extract DummyPage component with its styles into the polymath-ui at all